src/main/resources/format.yaml

```diff
@@ -4,6 +4,7 @@ statusUrl: "https://github.com/xstefank/test-repo/blob/master/.github/pull_reque
 
 format:
   title: "\\[WFLY-\\d+\\]\\s+.*|WFLY-\\d+\\s+.*"
+# commit: "regex"
```

Comment:
why this needs to be commented?

Reply:
I think this format is used like a template or not? I want to show that you can set for commit different regex.

Reply:
because by default it uses regex from title.

Reply:
so if you want to showcase this option, this line shouldn't be present, this comment doesn't add any value :)